stdlib/stomp/src/main/ballerina/stomp/sender.bal

```diff
@@ -0,0 +1,174 @@
+import ballerina/socket;
+import ballerina/io;
+import ballerina/log;
+import ballerina/system;
+
+# Configurations related to a STOMP connection
+#
+# + host - STOMP provider url.
+# + port - STOMP port.
+# + config - Config.
+# + login - STOMP user login.
+# + passcode - STOMP passcode.
+# + vhost - default stomp vhost.
+# + acceptVersion - 1.1.
+# + socketClient - socketConnection.
+# + endOfFrame - null octet.
+
+public type Sender client object {
+
+    public string host = "";
+    public int port = 0;
```

Comment:
Shall we make host and port as mandatory parameters as we cannot make a connection without these? Also other fields that are needed